Author: Martin Pitt <mpitt@debian.org>
Description: Use Debian alternatives for external tools instead of hardcoded programs
Forwarded: No, Debian specific

Index: postgresql-9.0-9.0~beta1/src/bin/psql/print.h
===================================================================
--- postgresql-9.0-9.0~beta1.orig/src/bin/psql/print.h	2010-03-01 21:55:45.000000000 +0100
+++ postgresql-9.0-9.0~beta1/src/bin/psql/print.h	2010-05-01 15:01:10.477226993 +0200
@@ -172,7 +172,7 @@
 extern const printTextFormat *get_line_style(const printTableOpt *opt);
 
 #ifndef __CYGWIN__
-#define DEFAULT_PAGER "more"
+#define DEFAULT_PAGER "/usr/bin/pager"
 #else
 #define DEFAULT_PAGER "less"
 #endif
--- postgresql-8.3beta3/src/bin/psql/settings.h	2007-01-05 23:19:49.000000000 +0100
+++ postgresql-8.3beta3/src/bin/psql/settings.h	2007-11-21 17:10:38.000000000 +0100
@@ -19,7 +19,7 @@
 #if defined(WIN32) || defined(__CYGWIN__)
 #define DEFAULT_EDITOR	"notepad.exe"
 #else
-#define DEFAULT_EDITOR	"vi"
+#define DEFAULT_EDITOR	"sensible-editor"
 #endif
 
 #define DEFAULT_PROMPT1 "%/%R%# "
